packages/zoe/src/contracts/venueContract.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-use-before-define */
+// @ts-check
+
+import harden from '@agoric/harden';
+import produceIssuer from '@agoric/ertp';
+import { makeGetInstanceHandle } from '../clientSupport';
+
+/** @typedef {import('../zoe').ContractFacet} ContractFacet */
+
+// zcf is the Zoe Contract Facet, i.e. the contract-facing API of Zoe
+export const makeContract = harden(
+  /** @param {ContractFacet} zcf */ zcf => {
+    // Create the internal ticket mint
+    const { issuer, mint, amountMath: ticketAmountMath } = produceIssuer(
+      'operaTickets',
```

Comment:
almost everything else here is now agnostic to "opera tickets". could that too be provided by the creator of this?

Reply:
Good idea, I'll try to make this more general purpose